From 46d3d93b50c95437352615d1bcb0caaa315a79ec Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?St=C3=A9phane=20Marchesin?= <marcheu@chromium.org>
Date: Mon, 4 May 2015 14:55:19 -0700
Subject: [PATCH] CHROMIUM: st/mesa: Do not flush front buffer on context flush

Make gallium work again with new chrome.

BUG=none
TEST=compile

Change-Id: I9f37c62b15070d65680fce2576b79f13b41b7b05
Signed-off-by: James Ausmus <james.ausmus@intel.com>
Signed-off-by: Prince Agyeman <prince.agyeman@intel.com>
Signed-off-by: Dhinakaran Pandiyan <dhinakaran.pandiyan@intel.com>
Signed-off-by: Tomasz Figa <tfiga@chromium.org>
Signed-off-by: Deepak Sharma <deepak.sharma@amd.com>

Conflicts:
	src/mesa/state_tracker/st_manager.c
---
 src/mesa/state_tracker/st_manager.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/mesa/state_tracker/st_manager.c b/src/mesa/state_tracker/st_manager.c
index eebde62183..900d53008b 100644
--- a/src/mesa/state_tracker/st_manager.c
+++ b/src/mesa/state_tracker/st_manager.c
@@ -642,9 +642,6 @@ st_context_flush(struct st_context_iface *stctxi, unsigned flags,
       st->pipe->screen->fence_reference(st->pipe->screen, fence, NULL);
    }
 
-   if (flags & ST_FLUSH_FRONT)
-      st_manager_flush_frontbuffer(st);
-
    /* DRI3 changes the framebuffer after SwapBuffers, but we need to invoke
     * st_manager_validate_framebuffers to notice that.
     *
-- 
2.14.2

